templates/mod/geoip.conf.erb

```diff
@@ -0,0 +1,5 @@
+<%- if @enabled -%>
+GeoIPEnable On
```

Comment:
you could use the same as here https://github.com/igalic/puppetlabs-apache/blob/expires/templates/mod/expires.conf.erb#L1
```ruby
GeoIPEnable <%= scope.function_bool2httpd([@enabled]) %>
```